libraries/SITL/SIM_Submarine.h

```diff
@@ -45,9 +45,7 @@ class Submarine : public Aircraft {
 protected:
     const float water_density = 1023.6; // (kg/m^3) At a temperature of 25 °C, salinity of 35 g/kg and 1 atm pressure
 
-    const class FrameConfig {
-    public:
-        FrameConfig() = default;
+    struct FrameConfig {
```

Comment:
This is basically a revert of commit 767f81265ba1b905c40b7a129efb0a454e315971. I think a preferable fix would have been to do `frame_proprietary {}` but just reverting this will likely bring #7348 back (or worse, no compile error but the values set to 0).

Have you checked to see this is OK?

Also this is removing the const, which was a good thing to have.